fs_src/script.js

```diff
@@ -593,13 +606,18 @@ function updateComponent(cd) {
         el(c, "power_stats_container").style.display = "block";
       }
       if (cd.svc_type !== undefined) {
+        if (cd.hap_optional !== undefined && cd.hap_optional == 0) {
+          el(c, "svc_type_container").style.display = "none";
+        }
         selectIfNotModified(el(c, "svc_type"), cd.svc_type);
-        if (cd.svc_type == 3) {
-          selectIfNotModified(el(c, "valve_type"), cd.valve_type);
-          el(c, "valve_type_container").style.display = "block";
-          updateInnerText(el(c, "valve_type_label"), "Valve Type:");
-        } else {
-          el(c, "valve_type_container").style.display = "none";
+        if (cd.type != 11) {
+          if (cd.svc_type == 3) {
```

Comment:
```suggestion
          if (cd.svc_type == 3 || cd.svc_type == 4) {
```
We need the same for the W only mode

Reply:
careful this is svc_type not sys_mode, this does not what you think it does. everything is handled correctly through hap_optional flag

Reply:
Never mind, it's only the RGB controls with I can't disable in RGB+W mode and I think that's ok!

Reply:
oh jeah, thats a bug, I will fix that